### PR TITLE
Add script to display keyboard layout

### PR DESCRIPTION
**Pull request by bomgar**

### CODE REVIEW

File: key_layout/key_layout [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+setxkbmap -query | awk '/layout/{print $2}' | tr -d '\n'
+variant=$(setxkbmap -query | awk '/variant/{print $2}')
+if [ ! -z $variant ]; then
```
Comment (kb100) [inline key_layout/key_layout]: Please use `[[ ]]` (bash internal) instead of `[ ]` (separate process)

File: key_layout/key_layout [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  echo -n " "
+  echo $variant
```
Comment (kb100) [inline key_layout/key_layout]: These two lines could be consolidated into `echo " $variant"`, up to you. if you want to change.

**Comment by kb100** [comment]
Great idea for a blocklet! It should be ready to merge after addressing my concerns. If you make any new commits, please squash them into one for the merge, and amend your commit title to 

    key_layout: description of your script

**Comment by bomgar** [comment]
done

**Pull request merged**